jwst/datamodels/container.py

```diff
@@ -359,61 +315,47 @@ def from_asn(self, asn_data):
 
     def save(self,
              path=None,
-             dir_path=None,
              save_model_func=None,
              **kwargs):
         """
         Write out models in container to FITS or ASDF.
 
         Parameters
         ----------
-        path : str or func or None
+        path : str or None
             - If None, the `meta.filename` is used for each model.
             - If a string, the string is used as a root and an index is
               appended.
-            - If a function, the function takes the two arguments:
-              the value of model.meta.filename and the
-              `idx` index, returning constructed file name.
-
-        dir_path : str
-            Directory to write out files.  Defaults to current working dir.
-            If directory does not exist, it creates it.  Filenames are pulled
-            from `.meta.filename` of each datamodel in the container.
 
         save_model_func: func or None
             Alternate function to save each model instead of
             the models `save` method. Takes one argument, the model,
             and keyword argument `idx` for an index.
 
+        kwargs : dict
+            Additional parameters to be passed to the `save` method of each
+            model.
+
         Returns
         -------
         output_paths: [str[, ...]]
             List of output file paths of where the models were saved.
         """
         output_paths = []
-        if path is None:
-            def path(filename, idx=None):
-                return filename
-        elif not callable(path):
-            path = make_file_with_index
-
         for idx, model in enumerate(self):
-            if len(self) <= 1:
-                idx = None
+            print(model)
+            print(model.meta.filename)
+            print(save_model_func)
```

Comment:
Otherwise, this version of the save function looks fine to me - testing it with path, dir_path, and save_model_func options seems to work as expected.